### PR TITLE
Correct cache config definition.

### PR DESCRIPTION
**Pull request by johnou**

### CODE REVIEW

File: core/src/test/resources/test.conf [hunk 1]
```diff
@@ -67,7 +67,9 @@ primitives.foo {
 
 primitives.bar {
   type: set
-  cache.enabled: true
```
Comment (johnou) [inline core/src/test/resources/test.conf]: guess this syntax might work too.

Comment (kuujo) [reply core/src/test/resources/test.conf]: it works in Typesafe Config

Comment (kuujo) [reply core/src/test/resources/test.conf]: Should change these back. I'd like to keep them as examples for users. This test passes fine.

Comment (johnou) [reply core/src/test/resources/test.conf]: changed this one back, pretty sure the other one is broken and it's nice to show different syntax imo.

**Pull request merged**